operator/v1/types_network.go

```diff
@@ -256,6 +256,40 @@ type KuryrConfig struct {
 	// size by 1.
 	// +optional
 	OpenStackServiceNetwork string `json:"openStackServiceNetwork,omitempty"`
+
+	// enableDebug set to true will enable debug logs in kuryr-kuberntes.
+	// +optional
+	EnableDebug *bool `json:"enableDebug,omitempty"`
```

Comment:
you need to describe what happens if this is not set.  Also a bool pointer is weird unless you want to default ot true.

Reply:
I followed convention in the file, will change that.